src/connect.js

```diff
@@ -20,7 +20,8 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  const obj = params ? { ...params } : {};
+  return (extraParams) => extraParams ? { ...obj, ...extraParams } : obj;
```

Comment:
also, we can write like this, without checking
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```